tests/validate/ps_iops_test/src/ps_iops.cpp

```diff
@@ -104,18 +104,10 @@ runTestThread(const xrt::device& device, const xrt::kernel& hello_world,
 {
     std::vector<xrt::run> cmds;
 
-    auto bo0 = xrt::bo(device, DATA_SIZE, hello_world.group_id(0));
-    auto bo1 = xrt::bo(device, DATA_SIZE, hello_world.group_id(1));
-    auto bo0_map = bo0.map<int*>();
-    auto bo1_map = bo1.map<int*>();
-    std::fill(bo0_map, bo0_map + COUNT, 0);
-    std::fill(bo1_map, bo1_map + COUNT, 0);
-    bo0.sync(XCL_BO_SYNC_BO_TO_DEVICE, DATA_SIZE, 0);
-    bo1.sync(XCL_BO_SYNC_BO_TO_DEVICE, DATA_SIZE, 0);
     for (int i = 0; i < arg.queueLength; i++) {
         auto run = xrt::run(hello_world);
-        run.set_arg(0, bo0);
-        run.set_arg(1, bo1);
+        run.set_arg(0, xrt::bo(device, DATA_SIZE, hello_world.group_id(0)));
+        run.set_arg(1, xrt::bo(device, DATA_SIZE, hello_world.group_id(1)));
```

Comment:
This will not work. Setting an argument BO does not retain the BO (need to update documentation to this fact). 
 Only the BO internal data (address, size, type, etc) is used.  This means that the local xrt::bo object created on line 109 is destructed immediately at the semi-colon and the internal BO values that were copied are now invalid (dangling).